.github/workflows/run-test-cases.yml

```diff
@@ -1,4 +1,4 @@
-name: Run End-to-End Scenario
+name: Validate Akri with K3s, Kubernetes, and MicroK8s
```

Comment:
Should we also change where the "Run End-to-End Test" badge on the base readme points?

Reply:
was always the plan ... just had to know if/how github encoded the commas.